_config.yml

```diff
@@ -1,11 +1,17 @@
-remote_theme: pmarsceill/just-the-docs
 title: Espresso Aficionados
 description: Info from the Espresso Aficionados Discord server
-color_scheme: "light"
+
 aux_links:
   "r/espresso on Reddit":
     "//www.reddit.com/r/espresso/"
   "Espresso Aficionados Discord":
     "//discord.gg/espresso"
 aux_links_new_tab: true
-footer_content: "Copyright &copy; 2021 Espresso Aficionados"
+
+remote_theme: pmarsceill/just-the-docs
+color_scheme: "light"
+
+footer_content: "Copyright &copy; Espresso Aficionados"
```

Comment:
Note that I removed the year here. It's not necessary and saves us pesky updating on January 1st.